content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "activity center spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Activity Center
+
+## Designs
+
+- [designs](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5358%3A5666)
+
+## Use Cases
+
+TODO
+
+## Functional Requirements
+
+### General
+
+* The Activity Center MUST support filtering by 4 categories: "All", "Mentions", "Replies", "Contact Requests"
+* The Activity Center MAY contain the following type of items:
+    * contact request
+    * mention in a community
+    * mention in public chat
+    * mention in a group
+    * mentions in a 1:1
+    * reply to a message from the user
+* Clicking the "Mark all as read" MUST mark all activity items read of the selected category
+* The activity items MUST be grouped by day sections and sorted by most recent
+* The day sections and their items MUST displayed according to the following criteria
+    * The section with items from today MUST be displayed as "Today"
+    * The section with items from yesterday MUST be displayed as "Yesterday"
+    * The section with items from this year MUST be displayed in the format "Weekday, Day Month" section example: "Mon, 21 April"
+    * The section with items from previous years MUST be displayed in the format "Day Month Year" section example: "Mon, 21 April 2020"
+* Clicking the context menu MUST display the following options:
```

Comment:
I assume this refers to the kybab ("..." menu icon) that was in the top right of the activity center.

This icon has been removed from the latest designs (see https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1267%3A320353 ) and replaced with an icon that only represents the "Hide/Show notifications that have been read" action.  The default state for this button should be "Show notifications that have been read".